src/server/journal/streamer.h

```diff
@@ -98,7 +98,18 @@ class RestoreStreamer : public JournalStreamer {
 
   // Returns whether anything was written
   void WriteBucket(PrimeTable::bucket_iterator it);
-  void WriteEntry(string_view key, const PrimeValue& pk, const PrimeValue& pv, uint64_t expire_ms);
+  void WriteEntry(std::string_view key, const PrimeValue& pk, const PrimeValue& pv,
+                  uint64_t expire_ms);
+  void WriteCommand(std::string_view cmd, absl::Span<const std::string_view> args);
```

Comment:
I think this new funciton should be seperated to a new class which will be in a new file
This class will handle serialization of entries for migration, similar to what we have with rdb serializer

Reply:
When you write "this new function" you only mean `WriteCommand()`, or something broader than that?